### PR TITLE
dub upgrade

### PR DESCRIPTION
**Pull request by John-Colvin**

### CODE REVIEW

File: examples/phobos/create.d [hunk 1]
```diff
@@ -75,13 +75,13 @@ void run(string[] args) @safe {
...
-                    "pyd": "0.13.0",
-                    "unit-threaded": "0.10.5"
-                }
+            	"fileVersion": 1,
+            	"versions": {
+            		"autowrap": {"path":"../../.."},
+            		"mirror": "0.3.0",
+            		"pyd": "0.13.1",
+            		"unit-threaded": "1.0.4"
+            	}
```
Comment (John-Colvin) [inline examples/phobos/create.d:84]: changed formatting to match what `dub upgrade` produces.

File: examples/phobos/std_algorithm/dub.selections.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-        "pyd": "0.13.0",
-        "unit-threaded": "0.10.5"
-    }
+	"fileVersion": 1,
+	"versions": {
+		"autowrap": {"path":"../../.."},
+		"mirror": "0.3.0",
+		"pyd": "0.13.1",
+		"unit-threaded": "1.0.4"
+	}
```
Comment (John-Colvin) [inline examples/phobos/std_algorithm/dub.selections.json:8]: all these spaces->tabs is just because that's what `dub upgrade` produces.

**Comment by atilaneves** [comment]
I have a branch that updates mirror already since I need to in order to fix bugs wrapping carbon-d.

**Comment by John-Colvin** [comment]
rebased so it's now just the rest of the dub upgrades

**Pull request merged**